[DR-3119] Use new azure TF'ed secrets

nmalfroy opened the pull request:
There's a new application/set of secrets that are now Terraformed.  We should use those now

snf2ye commented on perf/datarepo/create-secret-manager-secret.yaml:
```diff
@@ -17,17 +17,17 @@ secrets:
       - kubeSecretKey: applicationsecret
-        path: secret/dsde/datarepo/perf/helm-azure
-        vaultKey: applicationsecret
+        path: secret/dsde/datarepo/dev/azure-application-secrets
+        vaultKey: client-secret
```
Same here, for perf?

samanehsan commented on integration/integration-2/create-secret-manager-secret.yaml:
```diff
@@ -17,17 +17,17 @@ secrets:
       - kubeSecretKey: applicationsecret
-        path: secret/dsde/datarepo/integration/helm-azure-integration
-        vaultKey: applicationsecret
+        path: secret/dsde/datarepo/dev/azure-application-secrets
+        vaultKey: client-secret
```
I'm wondering the same thing 🤔

nmalfroy replied:
There isn't.  So the challenge is that the terraform creates resources for the different environments as part of the secret creation but it felt like overkill given that we use a lot of dev resources in integration and perf already.  That being said, I'm happy to create those in the TF

snf2ye replied:
I think it's fine without it, I just wanted to make sure it was on purpose.

samanehsan commented:
Could you update the Data Repo set up docs to include the new variables we need to have set? https://github.com/DataBiosphere/jade-data-repo/blob/develop/docs/jade-getting-started.md#1-build-jade-data-repo

nmalfroy commented:
I updated the values for integration and perf with the new resources and secrets that I introduced here:
https://github.com/broadinstitute/terraform-ap-deployments/pull/1143

nmalfroy commented:
Ah, yeah, I'll update the render-configs script too! (Though FWIW, for local dev, most folks probably don't need to set the synapse initialization variable...)
